runbotics/runbotics-actions-windows/src/automation/office/excelError.logger.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export default class ExcelErrorLogger {
+    constructor() { }
+
+    static columnCoordinate(e?: Error): void {
+        throw new Error(`
+            ${this.disclaimer(e)}
+            Column must be a string (e.g. A).
+            Check the Input tab above.
+        `);
+    }
+
+    static startCellCoordinates(e?: Error): void {
+        throw new Error(`
+            ${this.disclaimer(e)}
+            Start row must be a number (e.g. 5).
+            Start column must be a string (e.g. A).
+            Check startRow and startColumn in Input tab above.
+        `);
+    }
+
+    static setCellIncorrectStructure(e?: Error): void {
+        throw new Error(`
+            ${this.disclaimer(e)}
+            Input must be an Array (e.g. [["C3", "D3", "E3"], ["C4", "D4", "E4"]]).
+            Check targetExcelStructure in Input tab above.
+            Try to pass it as variable (e.g. #{myArray}).
+        `);
+    }
+
+    static deleteColumnsIncorrectInput(e?: Error): void {
+        throw new Error(`
+            ${this.disclaimer(e)}
+            Input must be range or Array (e.g. "D", "C:E" or ["C", "D", "E"]).
```

Comment:
a lot of options. I'm not sure if not too much

Reply:
what do you suggest then? removing array option?